executor/table_reader.go

```diff
@@ -215,6 +219,7 @@ func (e *TableReaderExecutor) buildResp(ctx context.Context, ranges []*ranger.Ra
 		SetFromSessionVars(e.ctx.GetSessionVars()).
 		SetMemTracker(e.memTracker).
 		SetStoreType(e.storeType).
+		SetSchemaVer(txn).
```

Comment:
`SchemaVer` is stored in the SessionVars.TxnCtx.SchemaVersion.
https://github.com/pingcap/tidb/blob/master/session/session.go#L1945
 @lzmhhh123